### DISCUSSION TITLE
Is there a way to remove non-key fields for a Base ?

### DISCUSSION THREAD

**Discussion by xu2xulim** [deta/cloud-docs#455] [Help] [answered]
I have 3 fields that was created which I do not want. This applies to all items in the base. Is there a way to delete and remove the fields to keep only a finalise list of fields for that Base. Thanks

**Comment by jnsougata** [comment, marked as answer]
use the method `update` with `trim`
reference: https://docs.deta.sh/docs/base/sdk#update

**Reply by xu2xulim** [reply]
Would you be able to complete this puzzle


test.put({"f1" : "f1", "f2" : "f2"}) # Base has repeated number of same records but I really do not need any attribute f2
res = test.fetch()
for item in res.items :
    updates = {?} 
    test.update(?)

Thanks in advance.

**Reply by jnsougata** [reply]
? = `"f2": test.util.trim()`

**Reply by xu2xulim** [reply]
Thanks. Tested it and I works. 

It is there in words but an example would help a lot.
